### PR TITLE
bundleName_ in LLVMIRGen should get legalized.

### PR DESCRIPTION
**Pull request by navidR**
#3725

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -29,3 +29,6 @@ externalbackends/*/
 .vs*
 build_*/*
 .clangd/
+CMakeLists.txt.user
```
Comment (opti-mix) [inline .gitignore]: Please remove changes to .gitignore as they are unrelated.

**Comment by navidR** [comment]
Yeap, look at this :

```
 nm testBundle.o | c++filt 
0000000000000010 t libjit_stacked_kernel_0_specialized
0000000000000000 T .+_testMainEntry
0000000000000050 R .+_testMainEntry_config
0000000000000010 r .+_testMainEntrySymbolTable
```

I will update the pr to legalize the `mainEntryName_` too.

**Comment by opti-mix** [comment]
@navidR Have you tried building with -DGLOW_WITH_BUNDLES=ON and then running the bundles under `<your_build_dir>/bundles` and checking that they still run and produce correct results? We don't want to break them.

**Comment by opti-mix** [comment]
@navidR Some tests seem to be broken now. Could you please check what is the problem?

**Comment by navidR** [comment]
Yeap I noticed. I will try to fix the whole issue and will send you another patch (same pr).

**Comment by opti-mix** [comment]
@navidR Ping! Are you planning on landing this?

**Comment by navidR** [comment]
Yes, but this will take a little while since changing naming scheme changes a lot of things in Testing side. But if you have something ready go ahead. Because I encountered a lot of bugs in cross compilation process, so my priority is to fix them.

**Comment by mciprian13** [comment]
Any news on this?

**Comment by opti-mix** [comment]
Solved in #3901. Closing.

**Pull request closed without merging**